packages/web-forms/src/components/common/map/useMapViewControls.ts

```diff
@@ -243,8 +243,13 @@ export function useMapViewControls(mapInstance: Map): UseMapViewControls {
 				return;
 			}
 
-			const parsedCoords = fromLonLat([newLocation.longitude, newLocation.latitude]);
-			userCurrentLocationFeature.value = new Feature({ geometry: new Point(parsedCoords) });
+			const parsedCoords = fromLonLat([
+				newLocation.longitude,
+				newLocation.latitude,
+				newLocation.altitude ?? 0,
```

Comment:
This feels like a lie - if we don't know the altitude then leaving it undefined makes that clear. Does this cause an issue in OL?